src/networkcontroller/network_controller_test.go

```diff
@@ -100,88 +100,93 @@ func (suite *NetworkControllerTestSuite) TestNew() {
 }
 
 func (suite *NetworkControllerTestSuite) TestCreateNetwork() {
-	//Parameters
-	eth1 := entity.PhysicalPort{
-		Name:     suite.ifName,
-		MTU:      1500,
-		VlanTags: []int32{2043, 2143, 2243},
-	}
-
 	tName := namesgenerator.GetRandomName(0)
 	network := entity.Network{
-		Name: tName,
-		OVS: entity.OVSNetwork{
-			BridgeName:    tName,
-			PhysicalPorts: []entity.PhysicalPort{eth1},
+		Type:       entity.OVSKernelspaceNetworkType,
+		Name:       tName,
+		VLANTags:   []int32{0, 2048, 4095},
```

Comment:
Need double check: you're using `4095`, not `4096`?

Reply:
yes! 4096 is a invalid number